library.properties

```diff
@@ -1,8 +1,9 @@
-name=arduino_TLC59116
-version=1.5
-author=Alan Grover
+name=TLC59116
+version=1.8
+author=Red Byer (original by: Alan Grover)
 maintainer=Alan Grover <awgrover@msen.com>
-sentence=Library for TI TLC59116 (and almost TLC59108) I2C Led Driver chip.
+sentence=Library for TI TLC59116 (and almost TLC59108) I2C Led Driver chip. MODIFIED FOR ARDUINO 1.5+ 
```

Comment:
I haven't looked at the library spec in ages. Is there a "version of arduino" tag?

Reply:
Nope.  Just checked.  We could put all sorts of #ifdef to check Arduino framework versions.  I tend to just move on (since <1.5.6 is ages ago).